vm_image/build.sh

```diff
@@ -57,6 +57,6 @@ echo " * IMAGE_SIZE = '$IMAGE_SIZE'"
 echo "Building 'Dockerfile.vmdata'..."
 docker buildx build -t "vmdata-temp:$TAG" -f Dockerfile.vmdata . | indent
 echo "Building NeonVM image..."
-"./$NEONVM_BUILDER_PATH" --src "vmdata-temp:$TAG" --dst "$REGISTRY/$NAME:$TAG" --size "$IMAGE_SIZE" | indent
+"./$NEONVM_BUILDER_PATH" --src "vmdata-temp:$TAG" --use-inittab --dst "$REGISTRY/$NAME:$TAG" --size "$IMAGE_SIZE" | indent
```

Comment:
is this orthogonal?

Reply:
Unfortunately no; `v0.4.5`'s inittab stuff was unintentionally a breaking change, so `v0.4.6` fixed the "breaking"-ness of it by putting the feature behind a flag.